src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8340,7 +8340,7 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-  '/action-constraints/{actionConstraintName}':
+  '/action-constraints/{parameterConstraintName}':
```

Comment:
Maybe the endpoint should be `/action-parameter-constraints/{...}`?  (Not a deal-breaker)

Reply:
I was thinking about that from the beginning. After all discussion we had it seems to be the most fitting endpoint name (and least confusing).
I'll change the name to `/action-parameter-constraints/{...}`